OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -252,6 +257,12 @@ public virtual void Activate(Order order)
 			if (power == null)
 				return;
 
+			if (!HasSufficientFunds(power))
+				return;
+
+			if (Manager.PlayerResources != null)
```

Comment:
Is `PlayerResources` ever going to be `null` given it is assigned in the ctor?
I suspect we want to check `.HasValue`.

Reply:
We use `TraitOrDefault`, so it can be assigned to `null` in the ctor.

Reply:
It is `IsValueCreated` instead of `HasValue` which is for nullable types, but otherwise you are correct. Thanks for the hint.